.github/workflows/build_unstable_image.yml

```diff
@@ -0,0 +1,42 @@
+name: Build and push image
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  IMG_REGISTRY_HOST: quay.io
+  IMG_REGISTRY_ORG: kuadrant
+  IMG_NAME: testsuite
+  IMG_TAGS: unstable
+
+jobs:
+  build:
+    name: Build and push image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: Build Image
+        id: build-image
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: ${{ env.IMG_NAME }}
+          tags: ${{ env.IMG_TAGS }}
+          layers: true
+          platforms: linux/amd64
+          containerfiles: |
+            ./Dockerfile
+      - name: Push Image
+        if: ${{ !env.ACT }}
+        id: push-to-quay
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: ${{ steps.build-image.outputs.image }}
+          tags: ${{ steps.build-image.outputs.tags }}
```

Comment:
I really don't like the idea of pushing directly to the floating tag. I'd rather see pushing to tag named after the commit hash and then pushing to IMG_TAGS to have a tag handle for each image instead of ambiguous floating tag. The same applies also for promote_to_latest.

Reply:
Why?

Reply:
To have unambiguous handle for each unstable build e.g. (in which image version the issue has appeared, what image version was used for your testing) this could be somehow resolved by using image digests BUT all images in a registry without a tag may be deleted at any time by a garbage collector. This might not matter much for unstable images but it would definitively matter for "promoted" images.

Reply:
We have always used floating tags, even in the 3scale days and I cannot remember any problem we ever had with them :) Besides the "unchanging" images will still exist, and releases are not going away, they are just not automatic and require more work and I would like them to be aligned with the project releases. 

This just releases builds a releases some images with minimal work required for anyone to try.

Reply:
Discussed in person and decided to implement it later together with aligning building process with engineering's one.